radarpipeline/io/reader.py

```diff
@@ -26,18 +64,27 @@ class SparkCSVDataReader(DataReader):
     def __init__(self, config: Dict, required_data: List[str], df_type: str = "pandas",
                  spark_config: Dict = {}):
         super().__init__(config)
+        self.source_formats = {
+            # RADAR_OLD: uid/variable/yyyymmdd_hh00.csv.gz
+            "RADAR_OLD": re.compile(r"""^[\w-]+/([\w]+)/
+                                ([\d_]+.csv.gz|schema-\1.json)""", re.X),
+            # RADAR_NEW: uid/variable/yyyymm/yyyymmdd.csv.gz
+            "RADAR_NEW": re.compile(r"""^[\w-]+/([\w]+)/
+                                    [\d]+/([\d]+.csv.gz$|schema-\1.json$)""", re.X),
+        }
         default_spark_config = {'spark.executor.instances': 6,
                                 'spark.driver.memory': '10G',
                                 'spark.executor.cores': 4,
                                 'spark.executor.memory': '10g',
                                 'spark.memory.offHeap.enabled': True,
                                 'spark.memory.offHeap.size': '20g',
-                                'spark.driver.maxResultSize': '0'}
-
+                                'spark.driver.maxResultSize': '0',
+                                'spark.log.level': "OFF"}
```

Comment:
do we need logging? Or is the intention that we flip this config as needed via `update(spark_config)`